docs/advanced-usage/disabling-translations.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Disabling translations
```

Comment:
Please rename this to something relevant 🙂

Reply:
@freekmurze Did this make it in and then get removed? I can't see any documentation about disabling translations in the v4 docs folder

Reply:
I don't think I removed it on purpose.